triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -505,7 +505,7 @@ trait TriggerServiceFixture
                 encodedDars,
                 jdbcConfig,
                 false,
-                Cli.DefaultTlsConfiguration,
+                config.tlsClientConfig,
```

Comment:
By default, if TLS is not enabled in testing fixture, `config.tlsClientConfig` ends up being the same as `Cli.DefaultTlsConfiguration`. By using `config.tlsClientConfig` directly here, we allow TLS configurations to be correctly defined using overrides `tlsConfig` values influenced by the `tlsEnable` flag.